src/containers/Tx/useGetTx.js

```diff
@@ -65,7 +66,7 @@ const useGetTx = txHash => {
 				clearTimeout(listTimeout[i]);
 			}
 			const txRpc = await getDataAsync(`https://rpc.orai.io/tx?hash=0x${txHash}`);
-			if (isTxFetchSuccess(txRpc) || isTestnet) {
+			if ((isTxFetchSuccess(txRpc) || isTestnet) && !JSON.stringify(txRpc?.data?.result?.tx_result?.log).includes(SMARTCONTRACT_IGNORE)) {
```

Comment:
Can we make it more clarity by adding a meaningful variable ? 
May be `const notInIgnoredContract = !JSON.stringify(txRpc?.data?.result?.tx_result?.log).includes(SMARTCONTRACT_IGNORE)` 

You know context better.

Reply:
Thank you for your contributions. i will updated it